README.md

```diff
@@ -8,23 +8,89 @@
 [![Reddit](https://img.shields.io/reddit/subreddit-subscribers/stregasgate?style=social)](https://www.reddit.com/r/stregasgate/)
 [![Discord](https://img.shields.io/discord/641809158051725322?label=Hang%20Out&logo=Discord&style=social)](https://discord.gg/5JdRJhD)
 
-This package brings Raylib to Swift.
-## Setup
-Raylib for Swift uses Swift's C interoperability and compiler to build Raylib from source code.
-So there's no setup required, just add the package as a dependency and start making games!
+This package brings [Raylib](https://www.raylib.com/) to [Swift](https://www.swift.org/).
+
+Raylib for Swift uses Swift's C interoperability and compiler to build Raylib from source code. Just add the package as a dependency (to your swift executable package) and start making games!
+
+## Requirements & Setup
+
+On Windows and macOS you only need Swift. For Linux, see [install required libraries](https://github.com/raysan5/raylib/wiki/Working-on-GNU-Linux#install-required-libraries).
+
+### Creating a new project
+
+1. Create a Swift package for your project.
+
+```Bash
+$ cd my/projects/folder/path
+$ mkdir My_Game
```

Comment:
Let's change all instances of My_Game to MyGame. It's just cleaner because Swift is going to name everything after that folder.